internal/handlers/handler_reset_password_step2.go

```diff
@@ -31,7 +31,7 @@ func ResetPasswordPost(ctx *middlewares.AutheliaCtx) {
 
 	if err != nil {
 		switch {
-		case strings.Contains(err.Error(), ldapPasswordComplexityCode):
+		case utils.IsStringInSliceContains(err.Error(), ldapPasswordComplexityErrors):
 			ctx.Error(fmt.Errorf("%s", err), ldapPasswordComplexityCode)
```

Comment:
```suggestion
                case utils.IsStringInSliceContains(err.Error(), ldapPasswordComplexityCodes):
                        ctx.Error(fmt.Errorf("%s", err), ldapPasswordComplexityCode)
		case utils.IsStringInSlice(err.Error(), ldapPasswordComplexityErrors):
			ctx.Error(fmt.Errorf("%s", err), ldapPasswordComplexityCode)
```

So this was a bit more what I had in mind. Mainly think this will mean we can match error codes (0000052D), which should capture most errors. But we can also match specific error messages exactly.